<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DGetSelection</class>
 <widget class="QDialog" name="DGetSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>报表制作</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QPushButton" name="okPushButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="dataType">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>75</y>
     <width>471</width>
     <height>48</height>
    </rect>
   </property>
   <property name="title">
    <string>数据类型</string>
   </property>
   <widget class="QCheckBox" name="curValcheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>工程值</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="mostValcheckBox">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>22</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最值</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="avgValcheckBox">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>22</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>平均值</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="overValcheckBox">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>22</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>越限值</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="sumValcheckBox">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>22</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>统计值</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mostVal">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>129</y>
     <width>471</width>
     <height>48</height>
    </rect>
   </property>
   <property name="title">
    <string>最值</string>
   </property>
   <widget class="QCheckBox" name="maxValcheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="maxValTimecheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>22</y>
      <width>107</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大值发生时间</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="minValcheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>22</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="minValTimecheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>22</y>
      <width>107</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小值发生时间</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="overVal">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>183</y>
     <width>471</width>
     <height>92</height>
    </rect>
   </property>
   <property name="title">
    <string>越限选项</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>441</width>
      <height>64</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="overUpCountcheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越上限次数</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="overUpTimecheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越上限时间</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="overDownDownCountRatecheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越下下限时间</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="overDownCountcheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越下限次数</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="overDownTimecheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越下限时间</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="overDownDownTimeRatecheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越下下限时间</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="OverUpUpCountcheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越上上限次数</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="overUpUpTimecheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>越上上限时间</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="normalRatecheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Compliance</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>472</width>
     <height>57</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>时间</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>起始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>结束</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="endDateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setPushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>设置</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>间隔</string>
      </property>
      <widget class="QComboBox" name="TimeComboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15分钟</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30分钟</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1小时</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2小时</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3小时</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>DGetSelection</receiver>
   <slot>slot_on_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>DGetSelection</receiver>
   <slot>slot_on_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mostValcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DGetSelection</receiver>
   <slot>slot_on_mostValChange(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overValcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DGetSelection</receiver>
   <slot>slot_on_overValChange(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_on_ok()</slot>
  <slot>slot_on_cancel()</slot>
  <slot>slot_on_mostValChange(bool)</slot>
  <slot>slot_on_overValChange(bool)</slot>
 </slots>
</ui>
